commit 60b9881e124a91fd32195394db1e9ded40d941bc (refs/changes/37/637/2)
Author: Chris Burroughs <chris.burroughs@joyent.com>
Date:   2016-10-11T17:40:29-04:00 (3 years ago)
    
    RELENG-612 prefer (git+)https urls
    Reviewed by: Orlando Vazquez <orlando@joyent.com>
    Reviewed by: Trent Mick <trent.mick@joyent.com>
    Approved by: Orlando Vazquez <orlando@joyent.com>
    Approved by: Trent Mick <trent.mick@joyent.com>

diff --git a/package.json b/package.json
index 6f37851..a44c1d5 100644
--- a/package.json
+++ b/package.json
@@ -5,7 +5,7 @@
   "author": "Joyent (joyent.com)",
   "private": true,
   "dependencies": {
-    "manta": "git+ssh://git@github.com:jclulow/node-manta.git#a07d8f821d14ff4f8d0007aa1c4c6895508d9d55"
+    "manta": "git+http://github.com/jclulow/node-manta.git#a07d8f821d14ff4f8d0007aa1c4c6895508d9d55"
   },
   "engines": {
     "node": ">=0.8"
